infra/terraform/modules/service/batch.tf

```diff
@@ -0,0 +1,134 @@
+
+module "batch" {
+  source  = "terraform-aws-modules/batch/aws"
+  version = "~> 2.0.0"
+
+  instance_iam_role_name        = "${var.environment}-batch-app-ecs-instance-role"
+  instance_iam_role_path        = "/batch/"
+  instance_iam_role_description = "IAM instance role/profile for AWS Batch ECS instance(s)"
+  instance_iam_role_tags = {
+    ModuleCreatedRole = "Yes"
+  }
```

Comment:
Minor but do these tags add anything?

Reply:
Looking at the module code, not really,  other than tagging the roles as created by the module